models/player.go

```diff
@@ -305,28 +365,28 @@ func GetPlayerMembershipDetails(db DB, gameID, publicID string) (map[string]inte
 		}
 
 		for _, detail := range details {
-			ma := nullOrBool(detail.MembershipApproved)
-			md := nullOrBool(detail.MembershipDenied)
-			mb := nullOrBool(detail.MembershipBanned)
-			mdel := !mb && detail.MembershipDeletedAt.Valid && detail.MembershipDeletedAt.Int64 > 0
+			membershipApproved := nullOrBool(detail.MembershipApproved)
+			membershipDenied := nullOrBool(detail.MembershipDenied)
+			membershipBanned := nullOrBool(detail.MembershipBanned)
+			membershipDeleted := !membershipBanned && detail.MembershipDeletedAt.Valid && detail.MembershipDeletedAt.Int64 > 0
```

Comment:
What do you think to rename these variables to: 

1. `approvedMembership` 
2. `deniedMembership`
3. `bannedMembership`
4. `deletedMembership`

This way they follow the English form of `Adjective` + `Noun`

Reply:
Great, I will do it